Bio/PDB/cealign.py

```diff
@@ -110,36 +110,70 @@ def align(self, structure, transform=True):
             raise PDBException(msg)
 
         # Run CEAlign
-        # CEAlign returns the best N paths, where each path is a pair of lists
-        # with aligned atom indices.
+        # CEAlign returns the best N paths, sorted descending by length,
+        # where each path is a pair of lists with aligned atom indices.
         paths = run_cealign(self.refcoord, coord, self.window_size, self.max_gap)
-        longest_length = len(paths[0][0])
         longest_paths = [
-            (tuple(pA), tuple(pB)) for pA, pB in paths if len(pA) == longest_length
+            path for path in paths if len(path[0][0]) == len(paths[0][0][0])
         ]
 
         # Iterate over paths and find the one that gives the lowest
         # corresponding RMSD. Use QCP to align the molecules.
-        best_rmsd, best_u = 1e6, None
-        for path in longest_paths:
+        aln = QCPSuperimposer()
```

Comment:
Yeah, so, following my comment above. If we return a struct that can be converted to a namedtuple, we can simply keep the best namedtuple instead of carrying these variables around. Does this make sense?

I found [this](https://code.saghul.net/2012/04/pystructsequence-creating-named-tuples-in-c/) as a starting point to build a namedtuple in C, which might be nicer?